gptables/test/test_wrappers.py

```diff
@@ -547,3 +546,57 @@ def test_make_table_of_contents(self, testbook, create_gptable_with_kwargs,
         exp_contentsheet.table = None
 
         assert got_contentsheet.__dict__ == exp_contentsheet.__dict__
+
+    def test_make_notesheet(self, testbook):
+        """
+        Test that creating a notes table sheet using `make_notesheet` generates
+        the same gptables.GPTable object as expected.
+        """
+        gp_workbook = testbook.wb
+        gp_workbook._annotations = [1, 2]
+        dummy_table = pd.DataFrame(data={"Note number":[1, 2], "Note text":[3, 4]})
```

Comment:
When making dummy data, it's good practice to keep data the same type, for example in this case, keep "Note text" as a string